lab01-util/src/test/java/by/artempvn/lab01/util/StringUtilsTest.java

```diff
@@ -0,0 +1,13 @@
+package by.artempvn.lab01.util;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+
+public class StringUtilsTest {
+	
+    @Test 
+    public void isPositiveNumberTest() {
+        boolean actual=StringUtils.isPositiveNumber("31");
+        assertTrue(actual);
```

Comment:
Could be simplified to:
```suggestion
        assertTrue(StringUtils.isPositiveNumber("31"));
```